### ISSUE TITLE
Fetal chylothorax 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7220]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://publications.aap.org/pediatrics/article-abstract/147/3/e2020011601/33244/Congenital-Chylothorax-and-Hydrops-Fetalis-A-Novel?redirectedFrom=fulltext?autologincheck=redirected

**Comment by pnrobinson** [comment]
Fetal chylothorax

Def: The presence of chyle (a type of lipid-rich lymph) in the pleural space (the space surrounding the lung)  as observed in a fetus.
PMID:10472400
PMID:28351595

Comment: Fetal chylothorax is associated with a high mortality rate. The most serious consequences of fetal chylothorax are pulmonary hypoplasia, congestive heart failure, and hydrops. It may occur as an isolated disorder or as a feature of diseases such as trisomy 21, Turner syndrome, and Noonan syndrome. Chylous pleural effusion impairs normal development of fetal lungs by pressure effects and can lead to pulmonary hypoplasia, mediastinal shift, and impaired circulation

**Comment by pnrobinson** [comment]
Structure

Fetal pleural effusion
-- fetal hydrothorax
-- fetal chylothorax

**Comment by pnrobinson** [comment]
added
